content/api/suppression-list.apib

```diff
@@ -110,25 +146,20 @@ Please note that in the unlikely scenario where your receive a HTTP 5xx level er
                 }
         }
 
-## Search [/suppression-list{?to,from,domain,cursor,limit,per_page,page,sources,types,description}]
-
-### Search for List Entries [GET]
+### Search for List Entries [GET /suppression-list{?to,from,domain,cursor,limit,per_page,page,sources,types,description}]
 
 Perform a filtered search for entries in your suppression list.
```

Comment:
Describe what gets returned. Something like "Returns an array of suppression objects."